be/src/vec/functions/round.h

```diff
@@ -630,9 +630,9 @@ struct Dispatcher {
 
             return col_res;
         } else {
-            throw doris::Exception(ErrorCode::INVALID_ARGUMENT,
-                                   "Unsupported column {} for function truncate",
-                                   const_col_general->get_name());
+            LOG(FATAL) << "__builtin_unreachable";
```

Comment:
Actually I prefer using Exception instead of `LOG(FATAL)`, because we have no reason to explicitly make BE crash unless this logical error will lead to data incomplete or other problem worse than cluster shut down.

Reply:
I saw vec_const part using fatal, should I change together?